JS/Lesson30/onder/my-app/src/components/Ex9/İndex.js

```diff
@@ -0,0 +1,24 @@
+import { useState } from 'react';
+import './index.css'
+
+export const Ex9 = () => {
+
+  const [backgroundColor, setBackgroundColor] = useState('violet');
+
+  function changeBackgroundtoPurle(){
```

Comment:
Normally, you want to make functions reusable as much as possible. I'd change the code like this:
```
function changeBackgroundColor(color){
    setBackgroundColor(color);
  }
....
<button className='color-btn' onClick={() => changeBackgroundColor('purple')}>purple</button>
        <button className='color-btn' onClick={changeBackgroundtColor('orange'}>orange</button>

```